### PR TITLE
SVGKit: Add XCode 10 automatic fixes.

### PR DESCRIPTION
**Pull request by Shachlan**
While the git diff seems like this changes indentation, it's an
artifact of converting tabs to spaces. The visual indentation stays
the same under LT standards.

### CODE REVIEW

**Comment by Shachlan** [comment]
@StatusReport, please assign Rouven to this PR.

File: Source/DOM classes/SVG-DOM/SVGRect.h [hunk 1]
```diff
@@ -27,7 +27,7 @@ typedef struct
  
  Unfortunately, the SVG Spec authors defined "uninitialized" and "values of zero" to mean differnet things, so we MUST preserve
  that difference! */
-SVGRect SVGRectUninitialized();
+SVGRect SVGRectUninitialized(void);
```
Comment (CodeBaseCamp) [inline Source/DOM classes/SVG-DOM/SVGRect.h:30]: Why is this necessary?

File: Source/Vendor/Lumberjack/DDTTYLogger.m [hunk 2]
```diff
@@ -822,12 +822,12 @@ - (id)init
 		calendar = [NSCalendar autoupdatingCurrentCalendar];
 		
 		calendarUnitFlags = 0;
-		calendarUnitFlags |= NSYearCalendarUnit;
-		calendarUnitFlags |= NSMonthCalendarUnit;
-		calendarUnitFlags |= NSDayCalendarUnit;
-		calendarUnitFlags |= NSHourCalendarUnit;
-		calendarUnitFlags |= NSMinuteCalendarUnit;
-		calendarUnitFlags |= NSSecondCalendarUnit;
+		calendarUnitFlags |= NSCalendarUnitYear;
```
Comment (CodeBaseCamp) [inline Source/Vendor/Lumberjack/DDTTYLogger.m:825]: These are still tabs, it seems.

**Review by CodeBaseCamp — CHANGES_REQUESTED** [review]
Round 1.

File: Source/DOM classes/SVG-DOM/SVGRect.h (hunk 1 above)
Comment (Shachlan) [reply Source/DOM classes/SVG-DOM/SVGRect.h:30]: “This function declaration is not a prototype” warning

File: Source/Vendor/Lumberjack/DDTTYLogger.m (hunk 2 above)
Comment (Shachlan) [reply Source/Vendor/Lumberjack/DDTTYLogger.m:825]: done.

**Comment by Shachlan** [comment]
finished round.

**Comment by Shachlan** [comment]
ping @CodeBaseCamp

**Comment by Shachlan** [comment]
ping @CodeBaseCamp

**Comment by CodeBaseCamp** [comment]
The blanks should be tabs in the SVGKit repo, or not?

**Comment by Shachlan** [comment]
In the original SVGKit repo? They're spaces, not tabs.

**Comment by CodeBaseCamp** [comment]
I'm confused. The diff shows tabs, see [here](https://github.com/Lightricks/SVGKit/pull/5/files#diff-1c5cc849e963e294f28caab003e39625L236) for example.

**Comment by Shachlan** [comment]
Sorry, you're correct. LT uses spaces, the original uses tabs.

**Comment by CodeBaseCamp** [comment]
Can you update the code to use tabs?

**Comment by CodeBaseCamp** [comment]
Fix and squash.

**Comment by Shachlan** [comment]
@CodeBaseCamp finished fixing.
